gems/pending/util/MiqSshUtilV2.rb

```diff
@@ -205,6 +205,12 @@ def temp_cmd_file(cmd)
     end
   end
 
+  def self.generate_ssh_keys
+    require 'sshkey'
+    keys = SSHKey.generate
```

Comment:
So, after thinking about this again, I'd be fine with that.  Extracting a method to a library could be something we do when we actually need to re-use it.  The question of where it belongs once it's extracted may be something we never have to actually answer if we never re-use it.  I like waiting until we have two or more use cases so we can put it in the right place for both use cases.  👍